File: samples/java/ShoppingMall/Server/Configuration/src/main/java/systems/zlink/samples/shoppingmall/server/configuration/SampleTimings.java
package systems.zlink.samples.shoppingmall.server.configuration;

import java.time.Duration;

public final class SampleTimings {
    public static final Duration HttpTimeout = Duration.ofSeconds(10);
    public static final Duration WorkflowTimeout = Duration.ofSeconds(30);
    public static final Duration RequestTimeout = Duration.ofSeconds(30);

    private SampleTimings() {}
}

File: samples/kotlin/ShoppingMall/Server/Configuration/src/main/kotlin/systems/zlink/samples/kotlin/shoppingmall/server/configuration/SampleTimings.kt
package systems.zlink.samples.kotlin.shoppingmall.server.configuration

import java.time.Duration

object SampleTimings {
    val FrameworkTimeout: Duration = Duration.ofSeconds(8)
    val RequestTimeout: Duration = Duration.ofSeconds(10)
    val WorkflowTimeout: Duration = Duration.ofSeconds(8)
    val ChannelRetryDelay: Duration = Duration.ofMillis(100)
    val PollDelay: Duration = Duration.ofMillis(100)
    const val MaxChannelAttempts = 80
}
